components/video-page/hooks/use-rough-cut.ts

```
@@ -0,0 +1,249 @@
+      }
+    },
+    [roughCut]
+  );
+
+  return {
+    roughCut,
+    cameras,
+    error,
+    isStarting,
+    isDownloading,
+    isPolling: intervalRef.current !== null,
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/components/video-page/hooks/use-rough-cut.ts:244:16
  242 |     isStarting,
  243 |     isDownloading,
> 244 |     isPolling: intervalRef.current !== null,
      |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot access ref value during render
  245 |     start,
  246 |     download,
  247 |     reset,

(react-hooks/refs)